ceph-releases/kraken/ubuntu/16.04/daemon/check_zombie_mons.py

```diff
@@ -11,19 +11,18 @@
 else:
     kubectl_command = 'kubectl get pods --namespace=${CLUSTER} -l daemon=mon -o template --template="{ {{range  \$i, \$v  := .items}} {{ if \$i}} , {{ end }} \\"{{\$v.metadata.name}}\\": \\"{{\$v.status.podIP}}\\" {{end}} }"'
 
-monmap_command = "ceph mon getmap > /tmp/monmap && monmaptool -f /tmp/monmap --print"
+monmap_command = "ceph --cluster=${CLUSTER} mon getmap > /tmp/monmap && monmaptool -f /tmp/monmap --print"
 
 
 def extract_mons_from_monmap():
-    monmap = subprocess.check_output(monmap_command, shell=True)
-    mons = {}
+    monmap =  subprocess.check_output(monmap_command, shell=True)
+    mons = { }
```

Comment:
must be done :)
this was due to a copy past from an other check_zombie_mons.py so i also change it in the other files